pkgs/tools/misc/osm2pgsql/default.nix

```diff
@@ -14,17 +14,18 @@
 , libosmium
 , protozero
 , rapidjson
+, testers
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
:heart: Thanks for making this derivation [more override-friendly](https://nixos.org/manual/nixpkgs/stable/#idm140737319949456).